base/src/main/java/com/mindera/skeletoid/generic/KeyboardUtils.java

```diff
@@ -46,4 +46,19 @@ public static void hideKeyboardFrom(Context context, View view) {
                 .getSystemService(Activity.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
     }
+    
+    /**
+     * Show keyboard. This can be called from everywhere (Fragments for example)
+     *
+     * @param context App context
+     */
+    public static void showKeyboard(Context context) {
+        if(context == null){
```

Comment:
I would add a space after "if" and after ...null)

Reply:
done